File: js/task3c/S3_PIPE_04.js
const data = ["10", "5", "abc", "7.5", "", "3"];
function isNumber(str) {
  return !isNaN(str) && str.trim() !== "";
}

const result = data
  .filter(isNumber)       
  .map(Number)              
  .map(x => x * 2)         
  .reduce((acc, x) => acc + x, 0); 

console.log(result); 

File: python/task3c/S3_PIPE_04.py
data = ["10", "5", "abc", "7.5", "", "3"]

def is_number(s):
    try:
        float(s)
        return s.strip() != ""
    except ValueError:
        return False

result = sum(float(x) * 2 for x in data if is_number(x))

print(result)
